README.md

```diff
@@ -201,6 +292,49 @@ File 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267
 	Observed filenames: 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267,eicar.com.txt,eicar.txt
 ```
 
+```bash
+$ polyswarm_api -o /tmp/test.txt search search metadata '{ "query": { "exists": {"field": "lief.libraries"}}}'
+$ cat /tmp/test.txt | more
+Found 1000 matches to the search query.
+Search results for namespace(query=namespace(exists=namespace(field='lief.libraries')))
+File 3a56238678560ca57c87cc08d0e912a7cfddaa65c449135d908590780a086232
+	File type: mimetype: application/x-dosexec, extended_info: PE32 executable (GUI) Intel 80386, for MS Windows, UPX compressed
+	SHA256: 3a56238678560ca57c87cc08d0e912a7cfddaa65c449135d908590780a086232
+	SHA1: 29629198ac945de358952337da1015a8252c11f1
+	MD5: e74b9320e6c15363fa30319af75ff3e5
+	First seen: Thu, 16 May 2019 14:26:26 GMT
+	Observed countries: ,PR
```

Comment:
the output of this value has a problem with the first comma ",". It should be omitted when there is only one value.

Reply:
Will investigate.

Reply:
Fixed a bug in the output.